subgraph/bin/networks.js

```diff
@@ -6,8 +6,11 @@ const networksConfig = require('@unlock-protocol/networks')
 
 const networkFilePath = path.join(__dirname, '..', 'networks.json')
 
+const networkName = (n) =>
+  n === 'polygon' ? 'matic' : n === 'arbitrum' ? 'arbitrum-one' : n
```

Comment:
That's annoying. Do you think we could update the network config rather by changing the `subgraph` property to an object that has `endpoint` and `networkName` ?

Reply:
yes possible. It will just require to change accross the board in locksmith etc. Also we may end a `endpointV2` then

Reply:
yeah I like that

Reply:
see https://github.com/unlock-protocol/unlock/pull/9709